### DISCUSSION TITLE
Sonarr 4

### DISCUSSION THREAD

**Discussion by pax0707** [sct/overseerr#3135] [Help]
Does it support widowmaker?
Getting "Failed to get language profiles".

**Comment by ovidiuc88** [comment]
Getting the same, even if it used to work some time ago; Not even sure when it broke because I did not notice until recently.

**Reply by ovidiuc88** [reply]
> Getting the same, even if it used to work some time ago; Not even sure when it broke because I did not notice until recently.

I would like to make an update on my problem: it was solved by changing the Network profile type to "**Private network**" from public.
